### ISSUE TITLE
Cap round with thresholds dont work on one side.

### ISSUE THREAD

**Issue by DrakeAnglin** [ashish-chopra/ngx-gauge#127]
when added a threshold to a gauge one side is "round" and the other is "butt"

I am using version 6.0.0

![image](https://user-images.githubusercontent.com/37958526/200697674-d403c04a-38f1-46b9-8ced-00054f4634b3.png)

**Comment by luizgununes** [comment]
I'm having this issue too.

**Comment by SaraColombel** [comment]
Yes me too. Can't find the solution

**Comment by SaraColombel** [comment]
Rectification : I have no idea why but if your lowest threshold is 0, then it will appears not round ... : 
Here if i put '0.1' for CO2, it will work correctly :/ but VOC no. Hope this could help ! @luizgununes

const thresholds = {
    [IPSOModel.CO2]: {
        '1700': { color: State.major.color },
        '1200': { color: State.minor.color },
        '0.1': { color: State.normal.color },
    },
    [IPSOModel.VOC]: {
        '1': { color: State.major.color },
        '0.1': { color: State.minor.color },
        '0': { color: State.normal.color },
    },
};

**Comment by luizgununes** [comment]
You're right, with 0.1 both edges are rounded, but my segments colors aren't appearing, it uses the default grey as background.

**Comment by luizgununes** [comment]
Oh, i realized that probably all the segments are at the start point, that's why it's rounded, but the colors of the segment will not appear.
